docs/samples/dashboards/README.md

```diff
@@ -91,7 +91,7 @@ To customize the dashboard:
 
 ### 📝 Notes
 
-- Dashboard is compatible with Kuadrant v1.3.0-rc2+
+- Dashboard is compatible with Kuadrant v1.3.1+
```

Comment:
Which dashboard do we mean here?

Reply:
Honestly not sure, I can probably remove this, the Kuadrant stuff was just a find/replace of all the 1.3.0

Reply:
I am sure that's global rename, and that's what reviews are for :) Maybe we just list Kuadrant to avoid confusing README